container/src/validation.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+
+def load_credentials():
```

Comment:
Use the dotenv package. https://pypi.org/project/python-dotenv/. Very simple to use:

from dotenv import load_dotenv

```
load_dotenv()  # take environment variables from .env.

# Code of your application, which uses environment variables (e.g. from `os.environ` or
# `os.getenv`) as if they came from the actual environment.
```